pallets/nfts/src/weights.rs

```diff
@@ -795,19 +799,21 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	/// Storage: `Nfts::ItemConfigOf` (r:2 w:0)
 	/// Proof: `Nfts::ItemConfigOf` (`max_values`: None, `max_size`: Some(48), added: 2523, mode: `MaxEncodedLen`)
 	/// Storage: `Nfts::AccountBalance` (r:2 w:2)
-	/// Proof: `Nfts::AccountBalance` (`max_values`: None, `max_size`: Some(72), added: 2547, mode: `MaxEncodedLen`)
+	/// Proof: `Nfts::AccountBalance` (`max_values`: None, `max_size`: Some(120), added: 2595, mode: `MaxEncodedLen`)
+	/// Storage: `System::Account` (r:1 w:1)
```

Comment:
As above.

Reply:
Question: Interesting to see that there is only 1 read and 1 write to the `System::Account`. From my understanding, the `do_transfer` is called twice from A to B and then B to A so it must be 2 reads and 2 writes.